13.go

```diff
@@ -364,39 +364,37 @@ func (hs *serverHandshakeState) readClientFinished13(hasConfirmLock bool) error
 			return err
 		}
 
-		// 4.4.3: CertificateVerify MUST appear immediately after Certificate msg
-		msg, err = c.readHandshake()
-		if err != nil {
-			return err
-		}
+		if len(certs) > 0 {
```

Comment:
so this always true, isn't? hs.processCertsFromClient() is already checking if certificate has been sent and it will return an error if not

Reply:
Not if `ClientAuth` is set to `VerifyClientCertIfGiven`.